armi/physics/neutronics/fissionProductModel/fissionProductModelSettings.py

```diff
@@ -28,29 +29,127 @@ def defineSettings():
             CONF_FP_MODEL,
             default="infinitelyDilute",
             label="Fission Product Model",
-            description="The fission product model to use in this ARMI run",
+            description=(
+                "This setting is used to determine how fission products are treated in an analysis. "
+                "By choosing `noFissionProducts`, no fission products will be added. By selecting, `infinitelyDilute`, "
+                "lumped fission products will be initialized to a very small number on the blocks/components that require them. "
+                "By choosing `MO99`, the fission products will be represented only by Mo-99. This is a simplistic assumption that "
+                "is commonly used by fast reactor analyses in scoping calculations and is not necessarily a great assumption for "
+                "depletion evaluations. Finally, by choosing `explicitFissionProducts` the fission products will be added explicitly "
+                "to the blocks/components that are depletable. This is useful for detailed tracking of fission products."
+            ),
             options=[
                 "noFissionProducts",
                 "infinitelyDilute",
-                "2ndOrder",
-                "2ndOrderWithTransmutation",
                 "MO99",
+                "explicitFissionProducts",
+            ],
+        ),
+        setting.Setting(
+            CONF_FISSION_PRODUCT_LIBRARY_NAME,
+            default="",
+            label="Fission Product Library",
+            description=(
+                f"This setting is used when the `{CONF_FP_MODEL}` setting "
+                f"is set to `explicitFissionProducts` and is used to configure "
+                f"all the nuclides that should be modeled within the core. "
+                f"Setting this is equivalent to adding all nuclides in the "
+                f"selected code library (i.e., MC2-3) within the blueprints "
+                f"`nuclideFlags` to be [xs:true, burn:false]. This option acts "
+                f"as a short-cut so that analysts do not need to change their "
+                f"inputs when modifying the fission product treatment for "
+                f"calculations. This may be extended for other cross section "
+                f"generation codes."
```

Comment:
Yeah, I get in the habit of an all or nothing for f-strings. I can change it, but I like it?

Reply:
Hm, I actually assumed that making it an f-string would actually make it slower (even if it is just a teeny tiny bit). Turns out it doesn't impact the speed at all, since there is no internal arguments passed here.

Keep it then!